server/refishing-fst2.txt

```diff
@@ -8,13 +8,25 @@ clear
 push Dutch
 save stack dutch.bin
 
-# English
-define English [] ;
+# English (Needs fixing)
+def SK "s k " -> "ʃ ";
+def XS "x s " -> "k s ";
+def XH "x " -> "h ";
+def KN "k n " -> "n ";
+
+# hacks
+def AR "a r " -> "ɜ r ";
+def ER "ɜ r " -> "ɜː ";
+
+define English PGWord .o. SK .o. AR .o. ER;
 
 clear
 push English
 save stack english.bin
 
+# Should be able to do this, but it doesn't
+# f ɜː n / f ɜ r n / f a r n
+
```

Comment:
You can see the example hear which ought to work, and when we remove PGWord it does work, but for some reason it breaks with PGWord. You can try this out by just loading the whole English definition into foma with and without PGWord.

Reply:
So my question is: does it work with German? That did not work for me in the interactive way.

Reply:
It seems like it does. At the very least it doesn't have any errors running the german transducer, and my debugging shows that it's definitely running into errors on the English run (without PGWord) or it just gets nothing back (with PGWord).

Reply:
I haven't checked foma cli with this but I believe it does spit out `f a r n` with the German transducer in the current code.